boa_engine/src/vm/opcode.rs

```diff
@@ -517,13 +531,20 @@ pub enum Opcode {
     /// Stack: value, key, object **=>**
     SetPropertyByValue,
 
-    /// Defines a own property of an object by value.
+    /// Defines a class method by value.
     ///
     /// Operands:
     ///
     /// Stack: object, key, value **=>**
     DefineOwnPropertyByValue,
 
+    /// Defines a own property of an object by value.
+    ///
+    /// Operands:
+    ///
+    /// Stack: object, key, value **=>**
```

Comment:
I think the comments here are switched, right? The `DefineOwnPropertyByValue§. would define a "own property of an object", and the `DefineClassMethodByValue` would define the class method by value.